downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md

```diff
@@ -0,0 +1,400 @@
+- The `lang.xml:getContent()` function no longer accepts `xml:Text` as an argument. The new `lang.xml:data()` function can be used instead.
+- `lang.value` functions cannot be called on expressions of type `object`. Function call syntax (`value:func(obj)`) can be used instead.
+- The `lang.stream:reduce()` lang library function will only return the reduced value or an error (if the completion type includes a subtype of `error`). It no longer returns a nil `()` value. The return type of `lang.stream:reduce()` has been updated accordingly.
+- An issue causing the complement of numeric literals to return incorrect values has been fixed.
+- Negation has been disallowed with numeric literals.
+- Subtyping rules have been fixed to consider the `never` type, which denotes an empty set of values as a subtype of any type `T`.
+- Relational expressions with lists have been fixed to consider the relative order of lists with different sizes.
+- Any non-error value will match the wildcard match pattern at runtime even when the matched expression's static type includes a subtype of `error`.
+- An invalid attempt to transfer out a value via a return statement that uses `self` in an isolated object method results in a compilation error.
+- In a `readonly` and `class` intersection, the method qualifiers are now preserved from the class.
+
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha5%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+
+### Runtime Updates
+
+#### New Features
+
+###### Support for Configurable Variables with Records Having Fields of Record Types
+
+```ballerina
+public type Person readonly & record {
+     string name;
+     int id;
+     Address address;
+};
+
+public type Address  record {
+    string city;
+    County country;
+};
+
+public type County  record {
+    string name;
+};
+
+
+configurable Person person = ?;
+
+```
+
+The  `Config.toml` would be as follows.
+
+```toml
+[person]
+name = "waruna"
+id = 10
+address.city="Colombo"
+address.country.name="Sri Lanka"
+
+```
+
+###### Support for Configurable Variables with Arrays Having Fields of Record Types
+
+```ballerina
+configurable Person[] & readonly personArray = ?;
+```
+
+The  `Config.toml` would be as follows.
+
+```toml
+[[personArray]]
+name = "manu"
+id = 11
+address.city="New York"
+address.country.name="USA"
+
+[[personArray]]
+name = "hinduja"
+id = 12
+address.city="London"
+address.country.name="UK"
+```
+
+###### Support for Configurable Variables with Multidimentional Arrays
+
+```ballerina
+configurable int[][] & readonly int2DArr = ?;
+```
+
+The  `Config.toml` would be as follows.
+
+```toml
+int2DArr = [[1,2],[3,4]]
+```
+
+###### Support for Optional Module Name in TOML Syntax of Configurable Variables
```

Comment:
Shouldn't this be a level 5 heading? Check all.

Reply:
Fixed.